src/kernel/multitasking/tasks/task_small.c

```diff
@@ -1,32 +1,39 @@
 #include "task_small.h"
 
-#include <kernel/util/mutex/mutex.h>
-#include <kernel/segmentation/gdt_structures.h>
 #include <std/timer.h>
+#include <kernel/segmentation/gdt_structures.h>
 
-#define TASK_QUANTUM 20
+#define TASK_QUANTUM 10
+#define MAX_TASKS 64
 
 static int next_pid = 1;
 
-static task_small_t* _current_task = 0;
+task_small_t* _current_task_small = 0;
 static task_small_t* _task_list_head = 0;
+
 static timer_callback_t* pit_callback = 0;
+const uint32_t _task_context_offset = offsetof(struct task_small, machine_state);
```

Comment:
We record the offset of the machine_state field as it's the simplest way to be able to reference the offset from context_switch (which is in assembly)